### ISSUE TITLE
Not all renderers support fallback on Column

### ISSUE THREAD

**Issue by andrewleader** [microsoft/AdaptiveCards#3021]
<table>
<tr>
<th>Release</th>
<th>Renderer status</th>
<th>Tasks status</th>
</tr>
<tr>
<td>1.2.1
<td>✔️ .NET (#3208)<br/>✔️ Android (#3209)<br/>✔️ iOS (#3210)<br/>✔️ TS (#3211)<br/>✔️ UWP (#3212)</td>
<td>✔️ Shared (#3213)<br/>✔️ Designer (#3214)</td>
</tr>
</table>

In the spec (#2019), we stated that fallback should be on "all elements"... which is vague.

The following renderers **support** fallback on `Column`

* JS
* .NET WPF

The following renderers **do not support** fallback on `Column`

* UWP

I haven't tested the rest, I assume the rest are like UWP since they're using shared model

**DECISION**: Fallback on `Column` **should** be supported. The fallback **must** be another column (if it's not, issue parse warning and act as if fallback isn't set)

Repro card

```json
{
    "type": "AdaptiveCard",
    "body": [
        {
            "type": "ColumnSet",
            "columns": [
                {
                    "type": "Column",
                    "items": [
                        {
                            "type": "Image2",
                            "style": "Person",
                            "url": "https://pbs.twimg.com/profile_images/3647943215/d7f12830b3c17a5a9e4afcc370e3a37e_400x400.jpeg",
                            "size": "Small"
                        }
                    ],
                    "fallback": {
                        "type": "Column",
                        "items": [
                            {
                                "type": "TextBlock",
                                "text": "Fallback"
                            }
                        ]
                    },
                    "width": "auto"
                },
                {
                    "type": "Column",
                    "items": [
                        {
                            "type": "TextBlock",
                            "weight": "Bolder",
                            "text": "Matt Hidinger",
                            "wrap": true
                        },
                        {
                            "type": "TextBlock",
                            "spacing": "None",
                            "text": "Created {{DATE(2017-02-14T06:08:39Z,SHORT)}}",
                            "isSubtle": true,
                            "wrap": true
                        }
                    ],
                    "width": "stretch"
                }
            ]
        }
    ],
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "version": "1.0"
}
```

## JS renderer

![image](https://user-images.githubusercontent.com/13246069/58841239-d0fe2180-861d-11e9-9174-39cd74092bb5.png)

## UWP renderer

![image](https://user-images.githubusercontent.com/13246069/58841266-ec692c80-861d-11e9-9eb0-397c8b9b8966.png)

## WPF renderer

![image](https://user-images.githubusercontent.com/13246069/58841256-e3785b00-861d-11e9-9484-a55287a226f7.png)

# Auto-generated task status

- [x] Shared
- [x] Designer
- [x] .NET
- [x] Android
- [x] iOS
- [x] TS
- [x] UWP

**Comment by shalinijoshi19** [comment]
@RebeccaAnne thoughts? Is this something that we want to and can fix for 1.2.1 ? If so let's try to get on this next week!

**Comment by RebeccaAnne** [comment]
UWP fixed with #3197. Per Joseph iOS is dropping fallback columns and so still needs work. Not sure of the status of Android.

**Comment by shalinijoshi19** [comment]
Looks like this is fixed along with the children. Closing out.
